docs/content/configuration/v1alpha1-reference.md

```diff
@@ -0,0 +1,303 @@
+---
+title: v1alpha1 Reference
+date: 2019-10-04T19:40:55-07:00
+draft: false
+weight: 20
+menu:
+  docs:
+    parent: 'configuration'
+---
+
+Talos User Data is responsible for the host and Kubernetes configuration, and it is independent of other cloud config / cloud init data.
```

Comment:
```suggestion
Talos User Data is responsible for the host and Kubernetes configuration, and it is independent of other cloud init data.
```

Reply:
nit: A little unclear to me what it means to be "independent of other cloud config / cloud init data"